### DISCUSSION TITLE
How to avoid form validation in commandBar

### DISCUSSION THREAD

**Discussion by megaezz** [orchidsoftware/platform#2152] [Q&A]
I have a layout with form. Some fields are required. And I have 2 button in command bar: Save and Delete.
If I press Delete - page says me that I should fill required fields. But why I should, I just want to delete entity.
How can I avoid frontend validation when press delete?

**Comment by tabuna** [comment]
You can use the method to disable HTML validation when submitting forms:
```php
Button::make('Delete')
    ->method('delete')
    ->novalidate()
```

**Reply by megaezz** [reply]
I tried "novalidate".
Button now contains attribute formnovalidate="true". But form still validate.

**Reply by AlejandroAkbal** [reply]
Same issue as you
